### ISSUE TITLE
Update CONTRIBUTING.md for `staged.dependencies`

### ISSUE THREAD

**Issue by pawelru** [insightsengineering/nestdevs-tasks#89]
`staged.dependencies` had been replaced with a dedicated GitHubAction. Please update the contributing guidelines across our repos accordingly.

UDATE 03/2025
Let's change it into how to use `Remotes` field properly and remove the parts about monorepo and staged.deps.

**Comment by shajoezhu** [comment]
close? @pawelru

**Comment by pawelru** [comment]
Let me rephrase this to update for the `Remotes` field
